web_src/js/features/repo-mirror.js

```diff
@@ -0,0 +1,15 @@
+import $ from 'jquery';
+
+export function initMirrorRepoSyncUpdate() {
```

Comment:
Maybe this function is not needed?

![image](https://github.com/go-gitea/gitea/assets/2114189/40a803be-4a18-4504-97a6-cb6af4228479)

Reply:
I am trying to get rid of this function.

Reply:
@wxiaoguang I tried but there are multiple mirrors and they will be having id so that is blocking me to go implicit way as you have suggested.

Reply:
I didn't see any problem by doing so

bae26597ff4f233247c27c3a3eedc944c75670d7

Reply:
<img width="1383" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/b79a1556-cc7b-4043-8abc-ec474769e314">

This is the problem.
Dom is not happy.

Reply:
Oh, you made some fixes, let me check.

Reply:
Is it related to my code or is it from old code?

Reply:
Your and my code is okay. It was due to older code.

PS: Your solution works

Reply:
I am trying to find the diff in your code and mine. I got it now.